### PR TITLE
2 lab Matushonok 751002

### PR DESCRIPTION
**Pull request by pashamatushonok**

### CODE REVIEW

File: 2 lab Matushonok/index.html [hunk 1]
```diff
@@ -0,0 +1,85 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<title>���������</title>
+	<link rel="stylesheet" href="style.css">
+</head>
+<body>
+	<div class = "navigation">
```
Comment (pashamatushonok) [inline 2 lab Matushonok/index.html]: nav instead of  div

File: 2 lab Matushonok/index.html [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+			</ul>
+		</div>
+		<div class="small-navigation">
+			<p>NAVIGATION</p>
+			<div class="menu-icon">
+			<div></div>
+			<div></div>
+			<div></div>
+		</div>
+		<div style="clear: both; height: 0px;"></div>
```
Comment (pashamatushonok) [inline 2 lab Matushonok/index.html]: .clearfix  or some class

File: 2 lab Matushonok/index.html [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+		</div>
+	</div>
+
+	<div class="blue-rect">
+		<h1>Awesome Free PSD<br>
+		templates & Resources</h1>
+		<h2>www.BlazRobar.com</h2>
+		<p>></p>
```
Comment (pashamatushonok) [inline 2 lab Matushonok/index.html]: icon (font-awesome, content in :after/:before)

**Comment by aliaksandrkrasnou** [comment]
Approved, okay 👍

**Pull request closed without merging**